e2e/basic-smoke.spec.js

```diff
@@ -18,4 +18,12 @@ describe('Basic smoke tests', () => {
 		await element(by.id('header-back')).tap()
 		await expect(element(by.id('screen-homescreen'))).toBeVisible()
 	})
+
+	it('should show home screen after tap to exit edit-home screen', async () => {
```

Comment:
Please put this in a new file.

Reply:
This is part of the general homescreen smokescreen tests. Why?

Reply:
We're testing out the two top buttons here.

Reply:
It's part of the _homescreen_ smoke tests, not the _app_ smoke tests. This file is just to make sure that we can open the app and do a single navigation. Everything else should go into new files.

Reply:
"e2e/views/homescreen/edit.spec.js"?

And for now, just this one; in the future, things like testing that we can drag and drop the rows (not yet possible in Detox) or ensuring that flipping a toggle switch hides the view from the homescreen. Or, in e2e/views/homescreen/home.js, making sure that you can scroll to the bottom on smaller devices.